exporter.py

```
@@ -44,12 +44,33 @@ def extract_media(sentence_id: int, out_dir: str, pad_start: float = 0.5, pad_en
+                test_path = os.path.join(dir_name, v_name + ext)
+                if os.path.exists(test_path):
+                    found_video = test_path
+                    break
+            if found_video:
+                break
+
+        if found_video:
+            mkv_path = found_video
+        else:
+            # Fallback
+            mkv_path = base_name + ".mkv"
```

Ruff (0.16.2)

[warning] Use binary operator `mkv_path = found_video or base_name + ".mkv"` instead of `if`-`else`-block

(SIM108)